installer/bundle/bundle_skel.sh

```diff
@@ -1066,6 +1098,20 @@ if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
     compare_install_type
     compare_arch
 
+    python_installed
+    if [ $? -ne 0 ]; then
+        if [ -z "${forceFlag}" ]; then
+            echo "Error: Python is not installed on this system, installation cannot continue." >&2
+            echo "Please install either the python2 or python3 package." >&2
+            echo "You can run this shell bundle with --force; in this case, we will install omsagent," >&2
+            echo "but omsconfig (DSC configuration) will not be available and will need to be re-installed." >&2
```

Comment:
Will the omsagent work correctly (albeit without the configs) without DSC installed and won't crash because of some missing file that dsc needs to create that we might overlook??

Reply:
look below
this is duplicate behavior of what we do if python-ctypes is not detected

Reply:
Oh nvm. I didn't see the code below.